### PR TITLE
Dropping user account endpoints and functionality

### PR DESCRIPTION
**Pull request by FedorLap2006**
This PR drops all user account related functionality and endpoints.
Due to the fact it violates Discord ToS and due to [recent changes](https://github.com/discord/discord-api-docs/discussions/4456) in rules for Discord Developers Rommunity Resources it needs to be dropped.

### CODE REVIEW

File: discord.go [hunk 1]
```diff
@@ -75,87 +55,8 @@ func New(args ...interface{}) (s *Session, err error) {
 	s.Identify.Properties.OS = runtime.GOOS
 	s.Identify.Properties.Browser = "DiscordGo v" + VERSION
 	s.Identify.Intents = MakeIntent(IntentsAllWithoutPrivileged)
```
Comment (vidhanio) [inline discord.go:57]: if this is being refactored anyways, `MakeIntent` is a no-op and not needed to be used anymore.

Comment (FedorLap2006) [reply discord.go]: Done.

**Comment by bwmarrin** [comment]
I'm inclined to go forward to dropping user account exclusive code - but I'm open to discussion here and thoughts from anyone that thinks this will cause them any issues.  So we'll leave this open for a few days at least to allow for that.

**Comment by FedorLap2006** [comment]
Should be all good now!

**Pull request merged**